resources/startup.sh

```diff
@@ -21,6 +21,9 @@ function global_cfg_or_default {
   echo "${VALUE}"
 }
 
+# configure logging behaviour using the etcd property /cas/logging/root <ERROR,WARN,INFO,DEBUG>
```

Comment:
the key `/cas/logging/root` is a bit misleading. Since this is from inside the dogu we can just say `logging/root`